Helper/Constants.cs

```diff
@@ -5,12 +5,12 @@ namespace Helpers
     public class Constants
     {
         // Endpoints
-        public const string CONFIG_ADDRESS = "http://127.0.0.1:9005/v1";
+        public const string CONFIG_ADDRESS = "http://localhost:9005/v1";
         public const string TELEMETRY_ADDRESS = "http://127.0.0.1:9004/v1";
```

Comment:
Can you please convert all the addresses from 127.0.0.1 to localhost?

Reply:
Updated all to localhost. Need to make sure this still works fine in the test builds but it should.